CONTRIBUTING.md

```diff
@@ -14,9 +14,9 @@ from the VSCode marketplace. See
 To check out the repository, build from source, and install the VS Code plugin, make sure you have the following dependencies:
 
 - `rust`. Rust versions preceding 1.76.0 are not guaranteed to work. See [this web page](https://www.rust-lang.org/tools/install) for instructions on installing Rust.
-- `npm`. `npm` versions preceding 10.4.0 are not guaranteed to work.
+- `npm`. `npm` versions preceding 10.4.0 are not guaranteed to work. We are currently using npm v18.17.0.
 
-Then, run the following command:
+Then, run the following command to install all necessary dependencies:
```

Comment:
```suggestion
Then, run the following command:
```

"install all necessary dependencies" might make it seem like it installs `rust` and `npm`.